calcom/cal.com
# Cutting down calls for installed apps + other UI tweaks

## What does this PR do?

 - Cutting down installed apps page calls to just rely on a single call to installed apps of all variants.
 - Also, made sure the "Additional calendar" and "Destination calendar" dropdowns are showing correctly when the content gets too long. <img width="1185" alt="Installed_Apps___Cal_com" src="https://user-images.githubusercontent.com/467258/173386845-1c15e6eb-a77e-4935-a003-b331758adbeb.png">
 - Correcting CalDav Setup page component name

## Type of change




## How should this be tested?

Go to Installed App page and notice the time to load to have improved. Also, for dropdown fix, install a very long named calendar (radicale used in the above screenshot) to see it cropping the name. Couldn't make "..." ellipsis show within `react-select` without pseudo-class hacky ways, once I found a way I will create a follow-up PR tackling it.
